File: pset1/mario/mario.c
//this whole code creates a pyramid.
//you can only add in a value from 1 to 8.


#include <stdio.h>
#include <cs50.h>

int main(void){
    //define
    int height;

    do {
        height = get_int("Select a number from 1 to 8\n");
    }
    while (height > 8 || height < 1);

    for (int i = 1 ; i <= height; i++) {
        for (int k = i; k < height ; k++){
            printf(" ");

        }

        for (int j = 1; j <= i; j++) {
            printf("#");
        }


        printf("\n");
    }

}

File: PSET6 - PYTHON/mario/less/mario.py
from cs50 import get_int

height = 0

while (height <1 or height >8):

    height = get_int("Height: ")

if height >= 1 and height <= 8:

    for i in range(0, height, 1):

        for k in range(height-i-1, 0, -1):
            print(" ", end = "")

        for j in range(0, i+1, 1):
            print("#", end="")

        print()



